frontend/src/components/contact/Contact.js

```diff
@@ -0,0 +1,104 @@
+import React, { Component } from "react";
```

Comment:
This was moved to the `components/static_pages/contact.jsx` file; please pull the Carousel/splash image into that file.